mlflow/ml-package-versions.yml

```diff
@@ -49,6 +49,23 @@ pytorch:
     run: |
       pytest tests/pytorch/test_tensorboard_autolog.py
 
+lightning:
```

Comment:
We don't need an extra flavor, if lightning and pytorch_lightning are the same package, we just need to replace pytorch_lightning with lightning. While for backwards compatibility test, we could inside `run` pip uninstall lightning and install pytorch_lightning to run the test again.

Reply:
Removed extra flavor : c628e396173a4038402f7561b4b4cec3c0fa450e